src/components/shared/NavBar.js

```diff
@@ -4,12 +4,12 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 
 class NavBar extends Component {
     render() {
```

Comment:
Nothing changed. You can ignore this file!
I tried to make a navbar, but our routes in App.js to too complicated haha.